### PR TITLE
Add timeSlots informations

### PR DESCRIPTION
**Pull request by nicolasbarb**

### CODE REVIEW

File: SoWeQuiches/ViewModel/Login/LoginViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+//
+//  Created by Zakarya TOLBA on 15/02/2022.
+//
+
+import SwiftUI
+import Foundation
+import AuthenticationServices
+import RetroSwift
+
+class LoginViewModel: ObservableObject {
```
Comment (MaxenceMottard) [inline SoWeQuiches/ViewModel/Login/LoginViewModel.swift:13]: remove `LoginViewModel.swift`

File: SoWeQuiches/ViewModel/Home/HomeViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+//  LoginViewModel.swift
+//  SoWeQuiche
+//
+//  Created by Zakarya TOLBA on 15/02/2022.
+//
+
+import SwiftUI
+import RetroSwift
+
+class HomeViewModel: ObservableObject {
```
Comment (MaxenceMottard) [inline SoWeQuiches/ViewModel/Home/HomeViewModel.swift:11]: remove `HomeViewModel.swift`

File: SoWeQuiches/Views/Sign/HomeView+Logic.swift [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+             print(error)
+         }
+     }
+
+     func disconnect() async {
+         await applicationState.disconnect()
+     }
+
+     private func fetchActualTimeSlot() {
+
```
Comment (MaxenceMottard) [inline SoWeQuiches/Views/Sign/HomeView+Logic.swift:28]: remove

File: SoWeQuiches/Views/Sign/HomeView+Logic.swift [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+         actualTimeSlot = userTimeSlots.first {
+             if let startDate = $0.startDate {
+                 return startDate.timeIntervalSinceNow < 0
+             }
+             return false
+             }
```
Comment (MaxenceMottard) [inline SoWeQuiches/Views/Sign/HomeView+Logic.swift:34]: indent

File: SoWeQuiches/Views/Sign/HomeView+Logic.swift [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+     }
+
+     private func fetchNextTimeSlots() {
+         nextTimeSlots = userTimeSlots.compactMap {
+             guard let endDate = $0.endDate, let nextStartDate = actualTimeSlot?.startDate else { return nil }
+             return endDate > nextStartDate ? $0 : nil
+         }
+
+         print(nextTimeSlots)
```
Comment (MaxenceMottard) [inline SoWeQuiches/Views/Sign/HomeView+Logic.swift:43]: remove

File: SoWeQuiches/Views/Sign/HomeView+Logic.swift [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+             }
+             return false
+             }
+     }
+
+     private func fetchNextTimeSlots() {
+         nextTimeSlots = userTimeSlots.compactMap {
+             guard let endDate = $0.endDate, let nextStartDate = actualTimeSlot?.startDate else { return nil }
+             return endDate > nextStartDate ? $0 : nil
+         }
```
Comment (MaxenceMottard) [inline SoWeQuiches/Views/Sign/HomeView+Logic.swift:41]:
```suggestion
         nextTimeSlots = userTimeSlots.filter {
             guard let endDate = $0.endDate, let nextStartDate = actualTimeSlot?.startDate else { return false }

             return endDate > nextStartDate
         }
```

**Review by MaxenceMottard — CHANGES_REQUESTED** [review]

**Review by zaktolba — APPROVED** [review]

**Review by MaxenceMottard — APPROVED** [review]

**Pull request merged**